apis/v1alpha2/httproute_types.go

```diff
@@ -262,6 +262,7 @@ type HTTPPathMatch struct {
 	//
 	// +optional
 	// +kubebuilder:default="/"
+	// +kubebuilder:validation:MaxLength=1024
```

Comment:
apparently 2000 is the accepted folk limit around the web

Reply:
opinion:
Can you imagine anyone using a 2000 character path to do matching though?
If anything, I think we are already generous here.

Reply:
Maybe? There is data that 2000 is effectively the max which is well known. For the others, yes, we have less info so just have to pick a number. I'm not super tied to increase this -- just that we know that more about what the max is.